build/illume.js

```diff
@@ -11,17 +11,15 @@ var _q = require("@standard-library/q");
 var _kefir = require("kefir");
 
 var offsetAbove = function offsetAbove(y, element) {
-  return y > $(element).offset().top;
+  return y > element.offsetTop;
```

Comment:
`offsetTop` has a large caveat that jQuery works around: `offsetTop` only returns an offset relative to the parent element if it is positioned itself, rather than the offset to the document. jQuery's method is perhaps confusingly named, since it does something different than the native method.

This is prickly enough that it might be worth looking at small libraries that implement a more jQuery-like offset calculation.